barretenberg/cpp/src/barretenberg/stdlib/recursion/honk/verifier/protogalaxy_recursive_verifier.test.cpp

```diff
@@ -103,35 +86,43 @@ template <typename RecursiveFlavor> class ProtoGalaxyRecursiveTests : public tes
 
         big_a* big_b;
 
-        if constexpr (IsGoblinBuilder<InnerBuilder>) {
+        if constexpr (IsGoblinBuilder<Builder>) {
             auto p = point::one() * fr::random_element();
             auto scalar = fr::random_element();
             builder.queue_ecc_mul_accum(p, scalar);
             builder.queue_ecc_eq();
         }
     };
 
-    static std::shared_ptr<Instance> fold_and_verify_native(const std::vector<std::shared_ptr<Instance>>& instances,
-                                                            InnerComposer& composer)
+    static std::tuple<std::shared_ptr<ProverInstance>, std::shared_ptr<VerifierInstance>> fold_and_verify_native(
+        Composer& composer)
     {
-        auto folding_prover = composer.create_folding_prover(instances);
-        auto folding_verifier = composer.create_folding_verifier();
-
-        auto proof = folding_prover.fold_instances();
-        auto next_accumulator = proof.accumulator;
-        auto res = folding_verifier.verify_folding_proof(proof.folding_data);
-        EXPECT_EQ(res, true);
```

Comment:
What happened to this EXPECT_EQ?

Reply:
The folding verifier does not return true or false anymore, it returns \phi*. Previously it was returning true or false because it was checking the \phi* it computes against what the prover sent (via the transcript) and the prover was the one returning everything. Now we reflect what's happening in the paper more clearly.